framework/CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ Yii Framework 2 Change Log
 - Chg #9411: `DetailView` now automatically sets container tag ID in case it's not specified (samdark)
 - Chg #9953: `TimestampBehavior::getValue()` changed to make value processing consistent with `AttributeBehavior::getValue()` (silverfire)
 - New #10083: Added wrapper for PHP webserver (samdark)
+- Bug #10101: Fixed assignments saving on role removing (rezident1307)
```

Comment:
We usually order the changelog by type and issue id.
Could you move this line after the line for `Bug #10029`